sbnobj/Common/POTAccounting/BNBSpillInfo.h

```diff
@@ -6,30 +6,30 @@
  * @file sbnobj/Common/POTAccounting/BNBSpillInfo.h
  * @author Joseph Zennamo, FNAL (jaz8600@fnal.gov)
  */
-
 #include <vector>
 
 namespace sbn {
+
   class BNBSpillInfo {
   public:
-    //typedef enum triggertype { kBNB, kEXT, kUnbiased, kNUMI } TriggerType; //!< Which beamline are we looking at
-    
+
+    unsigned int event;
```

Comment:
This variable was moved from after `spill_time_ns` to here. This changes the layout of the class on disk.
While I think ROOT should be able to cope with this, it's better to keep things simple for it. Please move the `event` data member back to where it was (I guess now just before `FOM`).
Also note that this may change the checksum, so you may need to again perform the routine of manually update `classes_def.xml` to include only one new class version.